### PR TITLE
SCRUM-2459 Load Resource Descriptors file

### PR DESCRIPTION
**Pull request by markquintontulloch**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/interfaces/crud/ResourceDescriptorCrudInterface.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+@Path("/resourcedescriptor")
+@Tag(name = "CRUD - Resource Descriptor")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public interface ResourceDescriptorCrudInterface extends BaseIdCrudInterface<ResourceDescriptor> {
+
+	@GET
+	@Path("/{id}")
+	@JsonView(View.ResourceDescriptorView.class)
+	ObjectResponse<ResourceDescriptor> get(@PathParam("id") Long id);
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/interfaces/crud/ResourceDescriptorCrudInterface.java:28]: @markquintontulloch shouldn't need to define this as its already in the `BaseIdCrudInterface`

Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/interfaces/crud/ResourceDescriptorCrudInterface.java:28]: OH... unless you are overriding the view... sorry didn't catch that.

**Review by oblodgett — APPROVED** [review]

File: src/main/java/org/alliancegenome/curation_api/interfaces/crud/ResourceDescriptorCrudInterface.java (hunk 1 above)
Comment (markquintontulloch) [reply src/main/java/org/alliancegenome/curation_api/interfaces/crud/ResourceDescriptorCrudInterface.java:28]: yeah, missing the @Override though - will update

**Comment by oblodgett** [comment]
Not sure why the IT"s are failing but maybe this next run will work?

**Comment by markquintontulloch** [comment]
> Not sure why the IT"s are failing but maybe this next run will work?

Updates I made to the migration file got "lost" in a git stash

**Pull request merged**